src/libraries/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -9,7 +9,8 @@
     <Nullable>enable</Nullable>
     <IncludeInternalObsoleteAttribute>true</IncludeInternalObsoleteAttribute>
     <IsPackable>true</IsPackable>
-    <ServicingVersion>5</ServicingVersion>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
Thanks for adding the required OOB package authoring changes, here and in M.E.Logging.Abstractions.csproj.